pkg/apis/authentication/v1/defaults.go

```diff
@@ -17,9 +17,22 @@ limitations under the License.
 package v1
 
 import (
+	authenticationv1 "k8s.io/api/authentication/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
+
+var kubernetesServiceAud = []string{"https://kubernetes.default.svc"}
+
+func SetDefaults_TokenRequestSpec(obj *authenticationv1.TokenRequestSpec) {
+	if len(obj.Audiences) == 0 {
+		obj.Audiences = kubernetesServiceAud
```

Comment:
need a comment here detailing the guarantee around this default. I'd expect this default to continue issuing tokens valid against this API server. If we get external cluster identifiers in the future, it would probably make sense to either additionally include that, or switch solely to that.

Reply:
I think we would want to switch to the external cluster identifier solely. How do we want to handle this right now? If we do this defaulting in the registry we won't be able to version it, but we will be able to plumb through a configuration option.

Reply:
if the api audience is going to be configurable server-side (e.g. `--service-account-api-audiences`), does that mean we'd want someone requesting a token to be used against the API server to leave `audiences` empty?

If the audience for the apiserver isn't fixed and well-known, do we need another signifier in TokenRequest that means "give me a token valid for the apiserver", or is leaving audiences empty sufficient for that?

Reply:
Empty audience is probably enough. Let's default it in the registry.

Reply:
> Empty audience is probably enough. Let's default it in the registry.

sounds reasonable to me. add API field documentation that indicates it should be left empty to target the API server as an audience? would we ever want a token that could be presented to the API server *and* another audience? I think I'd want distinct tokens for that case.